autoload/ale/fixers/eslint.vim

```diff
@@ -50,10 +50,15 @@ function! ale#fixers#eslint#ApplyFixForVersion(buffer, version) abort
         return 0
     endif
 
+    let l:modules_dir = ale#path#FindNearestDirectory(a:buffer, 'node_modules')
+    let l:project_dir = !empty(l:modules_dir) ? fnamemodify(l:modules_dir, ':h:h') : ''
+    let l:cd_command = !empty(l:project_dir) ? ale#path#CdString(l:project_dir) : ''
+    let l:node_command = l:cd_command . ale#node#Executable(a:buffer, l:executable)
```

Comment:
this should not include cd_command if you're going to append it down on line 81 as well